docs/marionette.collectionviewadvanced.md

```diff
@@ -419,3 +421,32 @@ var MyCollectionView = Mn.CollectionView.extend({
   }
 });
 ```
+
+## Binding `ui`
+
+By default, `CollectionView` will not bind anything defined in `ui`. As it has
+no direct `template` of its own to manage, this isn't a particular issue.
+There may be instances where you want to bind the `ui` element to listen to
+children on the collection directly. For example, to bind `triggers` or
```

Comment:
Fair enough, though I'm not sure we should guarantee that behaviour in future development.